src/test/java/seedu/address/logic/parser/AddCommandParserTest.java

```diff
@@ -48,7 +48,7 @@
     @Test
     public void parse_allFieldsPresent_success() {
         Candidate expectedCandidate = new CandidateBuilder(BOB).withTags(VALID_TAG_FRIEND)
-                .withApplicationStatus("Pending").withInterviewStatus("Pending").build();
+                .withApplicationStatus("Pending").withInterviewStatus("Not Scheduled").build();
```

Comment:
Perhaps the `Pending` here could be replaced with a constant like `VALID_APPLICATION_STATUS`? Same for the interview status 👀

Reply:
thanks for pointing it out @domlimm ! Made the necessary changes 👍🏻

Reply:
😍